From 3451dbd813553d83d14ab1621570c59e09ae2772 Mon Sep 17 00:00:00 2001
From: arpio23 <alex@familyrutz.com>
Date: Sat, 11 Apr 2026 07:07:16 -0600
Subject: [PATCH] Revert "QEglPlatformContext: Also check for
 GL_EXT_ROBUSTNESS"

Description: Revert "QEglPlatformContext: Also check for
 GL_EXT_ROBUSTNESS"
 This breaks QtQuick rendering on hybris.

This reverts commit 30f2b5bc0b3fe851559f40dbc861b4c67203a4f4.
---
 .../opengl/platform/egl/qeglplatformcontext.cpp   | 15 +--------------
 1 file changed, 1 insertion(+), 14 deletions(-)

diff --git a/src/gui/opengl/platform/egl/qeglplatformcontext.cpp b/src/gui/opengl/platform/egl/qeglplatformcontext.cpp
index 350968b87d4..e56504833d1 100644
--- a/src/gui/opengl/platform/egl/qeglplatformcontext.cpp
+++ b/src/gui/opengl/platform/egl/qeglplatformcontext.cpp
@@ -104,14 +104,6 @@ QT_BEGIN_NAMESPACE
 #define GL_LOSE_CONTEXT_ON_RESET 0x8252
 #endif
 
-// Constants from GL_EXT_robustness.
-#ifndef GL_RESET_NOTIFICATION_STRATEGY_EXT
-#define GL_RESET_NOTIFICATION_STRATEGY_EXT 0x8256
-#endif
-#ifndef GL_LOSE_CONTEXT_ON_RESET_EXT
-#define GL_LOSE_CONTEXT_ON_RESET_EXT 0x8252
-#endif
-
 // Constants from EGL_NV_robustness_video_memory_purge
 #ifndef EGL_GENERATE_RESET_ON_VIDEO_MEMORY_PURGE_NV
 #define EGL_GENERATE_RESET_ON_VIDEO_MEMORY_PURGE_NV 0x334C
@@ -460,16 +452,11 @@ void QEGLPlatformContext::updateFormatFromGL()
                     }
                 }
             }
-            if (m_format.renderableType() == QSurfaceFormat::OpenGL && hasExtension("GL_ARB_robustness")) {
+            if (hasExtension("GL_ARB_robustness")) {
                 GLint value = 0;
                 glGetIntegerv(GL_RESET_NOTIFICATION_STRATEGY, &value);
                 if (value == GL_LOSE_CONTEXT_ON_RESET)
                     m_format.setOption(QSurfaceFormat::ResetNotification);
-            } else if (m_format.renderableType() == QSurfaceFormat::OpenGLES && hasExtension("GL_EXT_robustness")) {
-                GLint value = 0;
-                glGetIntegerv(GL_RESET_NOTIFICATION_STRATEGY_EXT, &value);
-                if (value == GL_LOSE_CONTEXT_ON_RESET_EXT)
-                    m_format.setOption(QSurfaceFormat::ResetNotification);
             }
         }
         runGLChecks();
-- 
2.53.0

